airbyte_cdk/sources/declarative/interpolation/macros.py

```diff
@@ -177,6 +177,7 @@ def format_datetime(
         dt_datetime = (
             datetime.datetime.strptime(dt, input_format) if input_format else str_to_datetime(dt)
         )
+    dt_datetime = dt_datetime.replace(tzinfo=pytz.utc)
```

Comment:
Outside of the scope of this PR. See https://airbytehq-team.slack.com/archives/C02U9R3AF37/p1743795253674379 for more details